spec/features/card_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+describe "Card", type: "feature" do
+   describe "adding" do
+     it "success adding card" do
+     visit new_card_path
+     within("#new_card") do
+       fill_in 'card_original_text', with: "Home"
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.